p2p/libp2p/directSender_test.go

```diff
@@ -69,7 +69,10 @@ func createLibP2PCredentialsDirectSender() (peer.ID, libp2pCrypto.PrivKey) {
 func TestNewDirectSender_NilContextShouldErr(t *testing.T) {
 	hs := &mock.ConnectableHostStub{}
 
-	ds, err := libp2p.NewDirectSender(nil, hs, func(msg p2p.MessageP2P) error {
+	ctx := context.Background()
```

Comment:
you can replace these 3 lines with a simple declaration

Reply:
done.